triggers/run-completion-event-trigger/proto/run_completion_event_trigger.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+
+package run_completion_event_trigger;
+
+option go_package = "github.com/sky-uk/kfp-operator/triggers/run-completion-event-trigger/proto/run-completion-event-trigger";
```

Comment:
What does this do?

Reply:
https://protobuf.dev/reference/go/go-generated/#package

OK I guess it's just needed by the Go gRPC generation tool

Reply:
https://protobuf.dev/reference/go/go-generated/#package